react-debugging.html

```diff
@@ -63,20 +69,17 @@
           padding: "10px"
         }
         // No need to change anything ABOVE this line ^
-
+        
         return (
           <h1>Draw a Card</h1>
-          <button onClick={}>
-            Click to Draw a Card
-          </button>
-          <button onclick={shuffle}>
-            Return Cards and Shuffle Deck
-          </button>
-          <div style={handStyles}>
+          <button> onClick= Click to Draw a Cardbutton></button>
+          <button> onClick={shuffle}>
+            Return Cards and Shuffle Deck></button>
+          <div> style={handStyles}>
             {hand.map((card, index) => {
               return (
-                <div style={cardStyles} key={index}>
-                  {card}
+                <div> style={cardStyles} key={index}
+                  {card}>
```

Comment:
Looks like there was a bit of struggle with this section. What process did you follow when evaluating the errors?